dtfs-central-api/api-tests/sql-db-helper.ts

```diff
@@ -7,7 +7,7 @@ import {
   UtilisationReportEntity,
 } from '@ukef/dtfs2-common';
 import { SqlDbDataSource } from '@ukef/dtfs2-common/sql-db-connection';
-import { aListOfZeroThresholdActivePaymentMatchingTolerances } from '../test-helpers/test-data/payment-matching-tolerances';
+import { aListOfZeroThresholdActivePaymentMatchingTolerances } from '../test-helpers';
```

Comment:
Can we please make this function names and variables more readable, other one looses the context behind length names.

Reply:
I'm not sure what you mean. Could you provide an example shorter name for this which maintains the same level of context for reference?

Reply:
Can this please be `activeZerotThresholdCurrencies` it defeats the purpose to actively name every single imperative operation a function is performing this is where doc string will help which also seems to be missing from the helper function `aListOfZeroThresholdActivePaymentMatchingTolerances` can this also please be added (appreciate this is not in your PR scope).

Reply:
As this test helper was originally added by @RLavender98, could I leave this with you?

Reply:
Going to resolve as beyond PR scope

Reply:
Can we please add a TODO, then I am happy mark it as resolved. Thanks

Reply:
Added TODO to the actual test helper file referencing [FN-3389](https://ukef-dtfs.atlassian.net/browse/FN-3389)